Vagrantfile

```diff
@@ -6,72 +6,23 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
+cd /home/vagrant/oppia
+apt-get update && apt-get install -y python-pip python-dev unzip
+bash ./scripts/setup.sh
+bash ./scripts/install_third_party.sh
+SCRIPT
```

Comment:
I think you are using the wrong script. The script with the dependencies is scripts/install_prerequisites.sh. 
This is also why you still needed to provide the dependencies yourself.

The scripts/install_third_party.sh gets then called automatically via scripts/start.sh

Could you please change that?